tooling/src/hypothesistooling/projects/hypothesispython.py

```diff
@@ -70,10 +70,10 @@ def build_docs(*, builder="html", only=()):
     # (unfortunately most options only have the short flag version)
     tools.scripts.pip_tool(
         "sphinx-build",
-        "-W",
-        "-T",
-        "-E",
-        "-b",
+        "--fail-on-warning",
+        "--show-traceback",
+        "--fresh-env",
+        "--builder",
```

Comment:
newer versions of sphinx added these long options (kind of amazing it took them until sphinx 8 to do so)